features/facebookImport/src/model/analyses/about-pictures-data-analysis.js

```diff
@@ -0,0 +1,62 @@
+import {
+    relevantZipEntries,
+    removeEntryPrefix,
+} from "../../importer/importer-util";
+import RootAnalysis from "./root-analysis";
+
+/**
+ * Minimum number of picutures that should be present in
+ * the export in order for this analysis to be active.
+ */
+const PICTURES_THRESHOLD = 3;
```

Comment:
Why 3? I guess one would be profile picture, another the "frontispice" picture (or whatever), then a third one?
Anyway, isn't it true that a single picture should trigger it, even if it's the profile picture?

Reply:
It is more or less a "random" value. There were discussions about when to show this (show it when there is a single picture, or a few more) but no concrete decision was taken. Hence a threshold to be able to experiment and change it if needed.

Reply:
My point is, what kind of behavior do we want to discourage here? Uploading *any* picture?